target/linux/ramips/dts/mt7621_dlink_covr-x1860-a1.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "mt7621.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	compatible = "dlink,covr-x1860-a1", "mediatek,mt7621-soc";
+	model = "D-Link COVR-X1860 A1";
+
+	aliases {
+		led-boot = &status_orange;
+		led-failsafe = &status_red;
+		led-running = &status_white;
+		led-upgrade = &status_red;
+	};
+
+	chosen {
+		bootargs = "console=ttyS0,115200";
+	};
+
+	keys {
+		compatible = "gpio-keys";
+
+		reset {
+			label = "reset";
+			linux,code = <KEY_RESTART>;
+			gpios = <&gpio 15 GPIO_ACTIVE_LOW>;
+		};
+
+		wps {
+			label = "wps";
+			linux,code = <KEY_WPS_BUTTON>;
+			gpios = <&gpio 8 GPIO_ACTIVE_LOW>;
+		};
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		status_white: power {
+			label = "white:status";
```

Comment:
perhaps add `color` and `function` attributes and drop `label`?

Reply:
is that how it's done upstream? I've never seen LEDs declared without label in OpenWrt

Reply:
`label` is deprecated: https://www.kernel.org/doc/Documentation/devicetree/bindings/leds/common.yaml

Reply:
But you should wait for #13837 before dropping the label property altogether.

Reply:
How are dependencies to another MR handled usaullay? Should we wait until the other MR is done to apply the changes?

Reply:
You can create a new pull request changing this after  https://github.com/openwrt/openwrt/pull/13837 was merged.